src/Command/Bugfix.php

```diff
@@ -60,16 +60,22 @@ public function __invoke(Environment $env, Arguments $arguments, Options $option
         $env->output()->write(Str::of("Next release: $newVersion\n"));
         $message = (new Question('message:'))($env->input(), $env->output());
 
+        $hasNoSignOption = $options->contains('no-sign');
+
         try {
             $message = new Message((string) $message);
         } catch (DomainException $e) {
-            $env->error()->write(Str::of("Invalid message\n"));
-            $env->exit(1);
+            if (!$hasNoSignOption) {
```

Comment:
```suggestion
            if (!$options->contains('no-sign')) {
```
I prefer having this call multiple times in this method since it better express the behaviour, and it removes the double negative introduced by the variable name